.github/workflows/test_containers.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         with:
           artifact_type: 'container'
           artifact_path: 'ubuntu:14.04'
+          platform: "arm64"
```

Comment:
Is this valid input? I thought OS is also required.

Reply:
Good catch, that is NOT valid input.
I'll fix that.